README.md

```diff
@@ -58,6 +59,7 @@ Options:
               "builder": "angular-cli-builders:custom-webpack-browser"
               "options": {
                      "webpackConfigPath": "./extra-webpack.config.js",
+                     "mergeStrategy": { "plugins": "prepend" },
```

Comment:
Would be better to have replace as example. Also would be nice to explain what happens in this particular example